server/routes/orders.js

```diff
@@ -157,14 +155,16 @@ router.post("/shipped", (req, res) => {
         var cur_index = 0;
         for (i = 0; i < results.length; i++) {
           if (results[i].stock < items_needed[i][1]) {
-            items_missing.push([results[i].itemId, items_needed[i][1] - results[i].stock]);
+            items_missing.push([
+              results[i].itemId,
+              items_needed[i][1] - results[i].stock
+            ]);
           }
         }
         if (items_missing.length !== 0) {
```

Comment:
@michaela-williams For future reference, when we need to do errors, you can set the status of the response to an error status. This will allow the client-side fetch to recognize it as an error and process it accordingly. That way you don't have to send empty data back on success.